components/alpaca/alpaca.app.mjs

```diff
@@ -1,11 +1,145 @@
+import { axios } from "@pipedream/platform";
+//Did not use Alpaca JS SDK because it does not work with OAuth
 export default {
   type: "app",
   app: "alpaca",
-  propDefinitions: {},
+  propDefinitions: {
+    isPaperAPI: {
+      type: "boolean",
+      label: "Paper API",
+      description: "Set to `true` if Paper API is used. Default is `false`.",
+      optional: true,
+    },
+    cancelOrders: {
+      type: "boolean",
+      label: "Cancel Orders",
+      description: "If `true` is specified, cancel all open orders before liquidating all positions.",
+      optional: true,
+    },
+    side: {
+      type: "string",
+      label: "Side",
+      description: "`buy` or `sell`",
+      options: [
+        "buy",
+        "sell",
+      ],
+    },
+    orderId: {
+      type: "string",
+      label: "Order Id",
+      description: "Order Id",
+      async options({ isPaperAPI }) {
+        const orders = await this.getOrders({
+          isPaperAPI,
+          params: {
+            limit: 500, //max allowed limit, no pagination
+          },
+        });
+        return orders.map((order) => ({
+          label: `${order.symbol} ${order.type} ${order.side} - ${order.qty ?? order.notional}`,
+          value: order.id,
+        }));
+      },
+    },
```

Comment:
See previous comment. I think you could use the `limit` and `after` properties here as well to paginate through the orders.